efi/crt0/crt0-efi-arm.S

```diff
@@ -191,3 +191,7 @@ _start:
 
 .L_DYNAMIC:
 	.word		_DYNAMIC - .
+
+#if defined(__ELF__) && defined(__linux__)
+	.section .note.GNU-stack,"",%progbits
+#endif
```

Comment:
do we have a newly implied version requirement for gnu-efi that we should guard against?

Reply:
Oooooh yes .. didn't even think
It is gnu-efi 3.0.15